users/richie/home/gui/firefox.nix

```
@@ -0,0 +1,292 @@
+{ pkgs, ... }:
+{
+  programs.firefox = {
+    enable = true;
+    profiles.richie = {
+      # extensions = with pkgs.inputs.firefox-addons; [
```

Using `pkgs.inputs.firefox-addons` only works in my config because of the overlay I have setup [here](https://gitlab.com/dkrautha/dotfiles/-/blob/ea9a05bdc1d198f8b5e9cff9912d816f595cbf1b/overlays/default.nix#L5). I think to get this to work you're going to need to pass inputs in up where `pkgs` is passed in, then you could do `inputs.firefox-addons` to access whatever you want. 

I believe I'm able to access inputs like this because I pass it as `specialArgs` when I generate my nixos system [here](https://gitlab.com/dkrautha/dotfiles/-/blob/8b95258f6494388bdc782ed38f89bc2ca45c35e1/flake.nix#L78). After digging through your flake.nix I found [this](https://github.com/RAD-Development/nix-dotfiles/blob/c9481499d1322112f98bb0075c072d28aa943298/lib/systems.nix#L163) where you generate your system configs, so I think using inputs should work fine.
```suggestion
      # extensions = with inputs.firefox-addons; [
```